crates/rolldown/src/stages/link_stage/tree_shaking.rs

```diff
@@ -206,7 +198,7 @@ fn include_member_expr_ref(ctx: &mut Context, symbol_ref: SymbolRef, props: &[Co
     // to the final access chains.
     map.insert(chains.into_boxed_slice(), (canonical_ref, cursor, namespace_property_name));
   }
-  // TODO(hyf0): suspicious
+  // https://github.com/rolldown/rolldown/blob/5fb31d0d254128825df9441b23da58e3f6663060/crates/rolldown/tests/esbuild/import_star/import_export_star_ambiguous_warning/entry.js#L2-L2
```

Comment:
you could reference this example https://github.com/rolldown/rolldown/blob/5fb31d0d254128825df9441b23da58e3f6663060/crates/rolldown/tests/esbuild/import_star/import_export_star_ambiguous_warning/entry.js#L2-L2